### DISCUSSION TITLE
测试iframe的集成方式报错，udify.app 拒绝了我们的连接请求。

### DISCUSSION THREAD

**Discussion by damoyang** [langgenius/dify#4368] [Help]
### html如下：
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>嵌入Iframe示例</title>
</head>
<body>
    <iframe
        src="https://udify.app/chatbot/pU2kDN6o5H3kZqYO"
        style="width: 100%; height: 500px; border: none;"
        allow="microphone">
    </iframe>
</body>
</html>

### udify.app 拒绝了我们的连接请求。
![image](https://github.com/langgenius/dify/assets/6905936/3c910424-7b05-473e-a728-e57b922a6a84)

**Comment by zuixiangaxuan** [comment]
同求，请问有答案了吗
